Formula/ansible.rb

```diff
@@ -682,8 +696,6 @@ def install
     Pathname.glob(libexec/"lib/python*/site-packages/prettytable-0.7.2-py*.egg-info").each do |prettytable_path|
       chmod_R("a+r", prettytable_path)
     end
-
-    man1.install Dir["docs/man/man1/*.1"]
```

Comment:
Why are man pages no longer installed? I've been using Ansible through Homebrew for years, but suddenly `man ansible-playbook` no longer works.

Reply:
Fix is in https://github.com/Homebrew/homebrew-core/pull/65365